com.locomalito.MalditaCastilla.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: com.locomalito.MalditaCastilla
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Platform
+command: maldita-castilla
+rename-desktop-file: maldita-castilla.desktop
+rename-icon: icon
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Sound access
+  - --socket=pulseaudio
+  # Direct Rendering access
+  - --device=dri
+modules:
+  - name: maldita-castilla
+    only-arches:
+      - i386
+    buildsystem: simple
+    build-commands:
+      - install -m755 -D maldita-castilla /app/bin/maldita-castilla
+      - install -m644 -D maldita-castilla.desktop /app/share/applications/maldita-castilla.desktop
+      - install -m644 -D assets/icon.png /app/share/icons/hicolor/64x64/apps/icon.png
+      - install -m644 -D icon.png /app/share/icons/hicolor/128x128/apps/icon.png
+      - install -m644 -D com.locomalito.MalditaCastilla.appdata.xml /app/share/appdata/com.locomalito.MalditaCastilla.appdata.xml
+      - mkdir -p /app/lib/maldita-castilla
+      - mv runner assets /app/lib/maldita-castilla
+    sources:
+      - type: archive
+        url: https://locomalito.com/juegos/Maldita_Castilla_linux.tgz
+        sha256: 9aa90d9ba7c1c40bde8be43214e5016455c9c024f1fe7774c5bbbd6ec8408376
+      - type: file
+        path: maldita-castilla.desktop
+      - type: file
+        path: icon.png
+      - type: script
+        dest-filename: maldita-castilla
+        commands:
+          - ln -sfT "$XDG_CONFIG_HOME" ~/.config
+          - exec /app/lib/maldita-castilla/runner "$@"
+      - type: file
+        path: com.locomalito.MalditaCastilla.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html